src/main/java/com/aws/greengrass/shadowmanager/configuration/ComponentConfiguration.java

```diff
@@ -14,12 +14,12 @@ public final class ComponentConfiguration {
     @Getter
```

Comment:
nit: Same, lets annotate the class itself with @Getter, instead of private members.